scripts/publish-root.gradle

```diff
@@ -29,6 +39,8 @@ nexusPublishing {
             stagingProfileId = sonatypeStagingProfileId
             username = ossrhUsername
             password = ossrhPassword
+            group = Configuration.artifactGroup
```

Comment:
We should remove this either here or in each module - I prefer it living in the module level config